CHANGELOG.md

```diff
@@ -37,6 +39,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 
 [Unreleased]: https://github.com/klaviyo/magento2-klaviyo/compare/1.2.4...HEAD
```

Comment:
Do we need to update this URL as well?

Reply:
yes we do, totally missed this one.. thanks!